### PR TITLE
Tabs: add a Getting Started tutorial

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: applications/GettingStartedWith/Tabs/index.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                template: '<div id="fourth">Fourth</div>'
+            }
+        ],
+        selectedIndex: 2,
+        onItemClick(e) {
+            showMessage(e.itemIndex + 1);
+        },
+        selectionMode: 'multiple'
+    });
+
```
Comment (EugeniyKiyashko) [inline applications/GettingStartedWith/Tabs/index.js]: empty line

File: concepts/05 UI Components/Tabs/00 Getting Started with Tabs/05 Create Predefined Tabs.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+                    text: 'Second',
+                    disabled: true                
+                },
+                {
+                    icon: 'favorites',
+                    text: 'Third'                
+                }
+            ]
+        });
+
```
Comment (EugeniyKiyashko) [inline concepts/05 UI Components/Tabs/00 Getting Started with Tabs/05 Create Predefined Tabs.md]: empty line

File: api-reference/10 UI Components/dxTabPanel/dxTabPanel.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ inherits: dxMultiView
 ---
 ---
 ##### shortDescription
-The TabPanel is a UI component consisting of the [Tabs](/concepts/05%20UI%20Components/Tabs/00%20Overview.md '/Documentation/Guide/UI_Components/Tabs/Overview/') and [MultiView](/concepts/05%20UI%20Components/MultiView/00%20Overview.md '/Documentation/Guide/UI_Components/MultiView/Overview/') UI components. It automatically synchronizes the selected tab with the currently displayed view and vice versa.
+The TabPanel is a UI component consisting of the [Tabs](/Documentation/ApiReference/UI_Components/dxTabs/) and [MultiView](/concepts/05%20UI%20Components/MultiView/00%20Overview.md '/Documentation/Guide/UI_Components/MultiView/Overview/') UI components. It automatically synchronizes the selected tab with the currently displayed view and vice versa.
```
Comment (albertov05) [inline api-reference/10 UI Components/dxTabPanel/dxTabPanel.md]:
```suggestion
The TabPanel is a UI component consisting of the [Tabs](/Documentation/ApiReference/UI_Components/dxTabs/) and [MultiView](/concepts/05%20UI%20Components/MultiView/00%20Overview.md '/Documentation/Guide/UI_Components/MultiView/Overview/') UI components. It automatically synchronizes the selected tab with the currently displayed view, and vice versa.
```

File: api-reference/10 UI Components/dxTabPanel/dxTabPanel.md [hunk 4]
```diff
@@ -6,7 +6,7 @@ inherits: dxMultiView
...
```
Comment (albertov05) [inline api-reference/10 UI Components/dxTabPanel/dxTabPanel.md:10]: I have a question regarding line 23 below.

Line 23:
text: "CA San Francisco Stanford Ave st."

Is "st" a typo?

File: api-reference/10 UI Components/dxTabs/dxTabs.md [hunk 5]
```diff
@@ -21,8 +21,8 @@ dx.web.js, dx.all.js
 #include common-demobutton with {
```
Comment (albertov05) [inline api-reference/10 UI Components/dxTabs/dxTabs.md:21]: I have a comment regarding line 9 above.

Line 9:
Can we update this line to the following?

`The Tabs component is a tab strip used to switch between pages or views. This UI component is included in the [TabPanel](/api-reference/10%20UI%20Components/dxTabPanel '/Documentation/ApiReference/UI_Components/dxTabPanel/') UI component, but you can use Tabs separately as well.`

File: concepts/05 UI Components/Tabs/00 Getting Started with Tabs/05 Create Predefined Tabs.md [hunk 6]
```diff
@@ -0,0 +1,163 @@
+You can create tab items in the [items](/Documentation/ApiReference/UI_Components/dxTabs/Configuration/items/) array, or populate tab items from a [dataSource](/Documentation/ApiReference/UI_Components/dxTabs/Configuration/#dataSource). This tutorial uses the first approach. To see how to use the [dataSource](/Documentation/ApiReference/UI_Components/dxTabs/Configuration/#dataSource), refer to the following demo: [Tabs Overview Demo](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Tabs/Overview/).
```
Comment (albertov05) [inline concepts/05 UI Components/Tabs/00 Getting Started with Tabs/05 Create Predefined Tabs.md]:
```suggestion
You can create tab items in the [items](/Documentation/ApiReference/UI_Components/dxTabs/Configuration/items/) array, or populate tab items from a [dataSource](/Documentation/ApiReference/UI_Components/dxTabs/Configuration/#dataSource). This tutorial uses the first technique. To see how to use the [dataSource](/Documentation/ApiReference/UI_Components/dxTabs/Configuration/#dataSource), refer to the following demo: [Tabs Overview Demo](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Tabs/Overview/).
```

File: concepts/05 UI Components/Tabs/00 Getting Started with Tabs/10 Create Custom Tabs.md [hunk 7]
```diff
@@ -0,0 +1,102 @@
+To apply customization to all items, use an [itemTemplate](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#itemTemplate). To customize an individual item, specify the [template](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/#template) property of the item.
```
Comment (albertov05) [inline concepts/05 UI Components/Tabs/00 Getting Started with Tabs/10 Create Custom Tabs.md]:
```suggestion
Use an [itemTemplate](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#itemTemplate) to apply customization to all items. To customize an individual item, specify the [template](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/#template) property of the item.
```

File: concepts/05 UI Components/Tabs/00 Getting Started with Tabs/10 Create Custom Tabs.md [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+
+Note that Angular and Vue use [custom templates](/Documentation/Guide/UI_Components/Common/Templates/#Custom_Templates) instead of the [template](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/#template) property. In React, specify the [render](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/#render) property.
+
+The following code adds a fourth custom tab.
```
Comment (albertov05) [inline concepts/05 UI Components/Tabs/00 Getting Started with Tabs/10 Create Custom Tabs.md]:
```suggestion
The following code adds a fourth custom tab:
```

File: concepts/05 UI Components/Tabs/00 Getting Started with Tabs/15 Specify Selection Mode.md [hunk 9]
```diff
@@ -0,0 +1,66 @@
+Users can select Tabs items in two different modes: `'single'` (default) or `'multiple'`. You can use the [selectionMode](/Documentation/ApiReference/UI_Components/dxTabs/Configuration/#selectionMode) property to change the mode. To preselect or to select an item programmatically, pass its index in the data source array to the [selectedIndex](/Documentation/ApiReference/UI_Components/dxTabs/Configuration/#selectedIndex) property. As an alternative, you can use the [selectedItem](/Documentation/ApiReference/UI_Components/dxTabs/Configuration/#selectedItem) (for `'single'` selection mode) or [selectedItems](/Documentation/ApiReference/UI_Components/dxTabs/Configuration/#selectedItems) (for `'multiple'` selection mode) properties.
```
Comment (albertov05) [inline concepts/05 UI Components/Tabs/00 Getting Started with Tabs/15 Specify Selection Mode.md]: `Users can select Tabs items in two different modes:`

Is "Tabs" the name of the component? If so, can we say "Tabs component items"? Or do you mean "Tab items"?

File: concepts/05 UI Components/Tabs/00 Getting Started with Tabs/15 Specify Selection Mode.md [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+
+The following code sets the [selectionMode](/Documentation/ApiReference/UI_Components/dxTabs/Configuration/#selectionMode) to `'multiple'` and preselects the third tab.
```
Comment (albertov05) [inline concepts/05 UI Components/Tabs/00 Getting Started with Tabs/15 Specify Selection Mode.md]:
```suggestion
The following code sets the [selectionMode](/Documentation/ApiReference/UI_Components/dxTabs/Configuration/#selectionMode) to `'multiple'` and preselects the third tab:
```

File: api-reference/10 UI Components/dxTabPanel/dxTabPanel.md (hunk 4 above)
Comment (vladaskorohodova) [reply api-reference/10 UI Components/dxTabPanel/dxTabPanel.md:10]: Yes, I guess it's a typo, I will change it.

File: api-reference/10 UI Components/dxTabs/dxTabs.md (hunk 5 above)
Comment (vladaskorohodova) [reply api-reference/10 UI Components/dxTabs/dxTabs.md:21]: Noted

File: concepts/05 UI Components/Tabs/00 Getting Started with Tabs/15 Specify Selection Mode.md (hunk 9 above)
Comment (vladaskorohodova) [reply concepts/05 UI Components/Tabs/00 Getting Started with Tabs/15 Specify Selection Mode.md]: I'll change it to "Tabs component items"

File: api-reference/10 UI Components/dxTabPanel/dxTabPanel.md (hunk 4 above)
Comment (albertov05) [reply api-reference/10 UI Components/dxTabPanel/dxTabPanel.md:10]: Thank you.

**Pull request merged**